Submitted By: Joe Ciccone <joeciccone@crazyeyesoft.com>
Date: 2005-11-23
Initial Package Version: 1.19
Upstream Status: Sent
Origin: Joe Ciccone <joeciccone@crazyeyesoft.com
Description: Adds flags to the stage1, stage2, and chain Makefiles
             forcing them to build as 32bit.
 
diff -Naur colo-1.19.orig/chain/Makefile colo-1.19/chain/Makefile
--- colo-1.19.orig/chain/Makefile	2004-10-26 10:27:01.000000000 +0000
+++ colo-1.19/chain/Makefile	2005-11-23 23:24:28.000000000 +0000
@@ -15,9 +15,9 @@
 
 include ../Rules.mak
 
-LDFLAGS= -nostdlib -Wl,-T,$(TARG).lnk,-Map,$(TARG).map
-CFLAGS= -Werror -Wall -Wstrict-prototypes -fomit-frame-pointer -O2 -fno-strict-aliasing $(CFLAGS_CPU) $(CFLAGS_COLO)
-CPPFLAGS= -nostdinc -Iinclude -I../include -D_DEBUG $(CPPFLAGS_GCC)
+LDFLAGS= -mabi=32 -nostdlib -Wl,-T,$(TARG).lnk,-Map,$(TARG).map
+CFLAGS= -mabi=32 -Wall -Wstrict-prototypes -fomit-frame-pointer -O2 -fno-strict-aliasing $(CFLAGS_CPU) $(CFLAGS_COLO)
+CPPFLAGS= -mabi=32 -nostdinc -Iinclude -I../include -D_DEBUG $(CPPFLAGS_GCC)
 
 all:
 	$(MAKE) -C .. binary
diff -Naur colo-1.19.orig/stage1/Makefile colo-1.19/stage1/Makefile
--- colo-1.19.orig/stage1/Makefile	2004-10-26 10:27:01.000000000 +0000
+++ colo-1.19/stage1/Makefile	2005-11-23 23:24:44.000000000 +0000
@@ -18,9 +18,9 @@
 
 include ../Rules.mak
 
-LDFLAGS= -nostdlib -Wl,-T,$(TARG).lnk,-Map,$(TARG).map
-CFLAGS= -Werror -Wall -Wstrict-prototypes -fomit-frame-pointer -O2 -fno-strict-aliasing $(CFLAGS_CPU) $(CFLAGS_COLO)
-CPPFLAGS= -nostdinc -Iinclude -I../include -D_DEBUG $(CPPFLAGS_GCC)
+LDFLAGS= -mabi=32 -nostdlib -Wl,-T,$(TARG).lnk,-Map,$(TARG).map
+CFLAGS= -mabi=32 -Wall -Wstrict-prototypes -fomit-frame-pointer -O2 -fno-strict-aliasing $(CFLAGS_CPU) $(CFLAGS_COLO)
+CPPFLAGS= -mabi=32 -nostdinc -Iinclude -I../include -D_DEBUG $(CPPFLAGS_GCC)
 
 all:
 	$(MAKE) -C .. binary
diff -Naur colo-1.19.orig/stage2/Makefile colo-1.19/stage2/Makefile
--- colo-1.19.orig/stage2/Makefile	2005-03-20 18:27:16.000000000 +0000
+++ colo-1.19/stage2/Makefile	2005-11-23 23:24:10.000000000 +0000
@@ -52,9 +52,9 @@
 
 include ../Rules.mak
 
-LDFLAGS= -nostdlib -Wl,-T,$(TARG).lnk,-Map,$(TARG).map,--emit-relocs
-CFLAGS= -Werror -Wall -Wstrict-prototypes -fomit-frame-pointer -O2 -fno-strict-aliasing $(CFLAGS_CPU) $(CFLAGS_COLO)
-CPPFLAGS= -nostdinc -Iinclude -I../include -D_DEBUG $(CPPFLAGS_GCC)
+LDFLAGS= -mabi=32 -nostdlib -Wl,-T,$(TARG).lnk,-Map,$(TARG).map,--emit-relocs
+CFLAGS= -mabi=32 -Wall -Wstrict-prototypes -fomit-frame-pointer -O2 -fno-strict-aliasing $(CFLAGS_CPU) $(CFLAGS_COLO)
+CPPFLAGS= -mabi=32 -nostdinc -Iinclude -I../include -D_DEBUG $(CPPFLAGS_GCC)
 
 LOADADDR= $(shell $(CROSS_COMPILE)readelf -l $(TARG) | awk '$$1 ~ /^LOAD$$/ { print $$3 }')
 
